docker/docker-compose-without-neo4j.override.yml

```diff
@@ -28,10 +28,17 @@ services:
     env_file: datahub-gms/env/docker-without-neo4j.env
     environment:
       - DATAHUB_TELEMETRY_ENABLED=${DATAHUB_TELEMETRY_ENABLED:-true}
+      - AUTH_POLICIES_ENABLED=${AUTH_POLICIES_ENABLED:-true}
```

Comment:
We should not need this here. We don't need every env variable supported by the container in the .yml. For most cases people will not use these so let's remove them from this file.

Reply:
Done

Reply:
Still here?

Reply:
removed from here.